### DISCUSSION TITLE
External deeplinks https redirect

### DISCUSSION THREAD

**Discussion by mrdrprofuroboros** [anyproto/contributors#264] [Anytype apps]
## Problem

Right now when you want to share a page within AnyType with your colleague, you can go to ... and use Copy Link

<img width="405" alt="Screenshot 2024-11-24 at 14 53 44" src="https://github.com/user-attachments/assets/652274c3-5270-4f85-a10a-694f44479378">

This gives you a deep link that looks like
```
anytype://object?objectId=<...>&spaceId=<...>
```
Unfortunately in most messengers this kind of custom deep links are not supported and don't get automatically highlighted, which makes it pretty annoying to manually copy and paste it to browser! (not even AnyType) to open

## Proposal

What if you host a simple https redirect service?
- once a user clicks "Copy Link" it generates an https link to the redirect service, say `https://link.anytype.io?objectId=<...>&spaceId=<...>`
- when you follow this link it just returns a deeplink redirect, the old `anytype://object?objectId=<...>&spaceId=<...>`
- user goes to anytype
- messengers highlight the link as usual, no fiddling
- Optional: for a self-hosted any-sync add the ability to redirect from it rather than `anytype.io`

**Comment by filip-anytype** [comment]
This is already planned for this season: https://github.com/orgs/anyproto/projects/1?pane=issue&itemId=85100917&issue=anyproto%7Croadmap%7C80.
